src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import '../../../../shared/css/nested-menu.scss'
+import '../../../AccountHeader/styles.scss'
+import '../../../AccountHeader/balance-selector.scss'
+import { localizeNumber, localizeBalance } from '../../../../shared/utils'
+import Currency from '../../../../shared/components/Currency'
+
+export interface AmmDataType {
+  balance: { currency: string; amount: number; issuer: string }
+  balance2: { currency: string; amount: number; issuer: string }
+  lpBalance: number
+  tradingFee: number
+  accountId: string
+  language: string
+}
+
+export const AMMAccountHeader = (props: AmmDataType) => {
+  const { balance, balance2, tradingFee, lpBalance, accountId, language } =
+    props
+  const { t } = useTranslation()
+  const b1 = localizeBalance(balance, language)
+  const b2 = localizeBalance(balance2, language)
+  const tradingFeeTotal = 1000
+  const lp = localizeNumber(lpBalance, language, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 2,
+  })
+  const tf = localizeNumber(tradingFee / tradingFeeTotal, language, {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 3,
+  })
```

Comment:
Can these variables be named more descriptively?

Reply:
Sure